<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Playerhistory</class>
 <widget class="QDialog" name="Playerhistory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QComboBox" name="gameComboBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>601</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun</family>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="return_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>340</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>SimSun</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color:#e3e3e3;
	color: #1323ff;
	border-style: solid;
	border-radius:15px;
	border-color: #000000;
	font-weight : bold;
	padding: 13px;

}


QPushButton::hover
{
	background-color: #3dffaf;
	color: #000000;
	border-style: solid;
	border-color: #000000;
	font-weight : bold;
	padding: 13px;

}


QPushButton::pressed
{
	background-color: #3dc8af;
	color: #000000;
	border-style: solid;
	border-color: #000000;
	font-weight : bold;
	padding: 13px;

}</string>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>291</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SimSun</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox 
{
	background-color: #00111d;
	border: 2px solid gray;


}


QGroupBox::title  
{
    background-color: #00111d;
    color: #ffffff;
    subcontrol-origin: margin;
    subcontrol-position: top;
}


QGroupBox::title::disabled
{
	background-color: #404040;
	color: #656565;

}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="name">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: transparent;
	color: #1323ff;
	border-color: #000000;

}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: transparent;
	color: #ffffff;

}


QLabel::disabled
{
	background-color: transparent;
	color: #656565;

}</string>
    </property>
    <property name="text">
     <string>Games Played</string>
    </property>
   </widget>
   <widget class="QLabel" name="gameplayedd">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun</family>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: transparent;
	color: #ffffff;

}


QLabel::disabled
{
	background-color: transparent;
	color: #656565;

}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: transparent;
	color: #ffffff;

}


QLabel::disabled
{
	background-color: transparent;
	color: #656565;

}</string>
    </property>
    <property name="text">
     <string>Games Won</string>
    </property>
   </widget>
   <widget class="QLabel" name="winsLabel">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: transparent;
	color: #ffffff;

}


QLabel::disabled
{
	background-color: transparent;
	color: #656565;

}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: transparent;
	color: #ffffff;

}


QLabel::disabled
{
	background-color: transparent;
	color: #656565;

}</string>
    </property>
    <property name="text">
     <string>Games Lose</string>
    </property>
   </widget>
   <widget class="QLabel" name="lossesLabel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: transparent;
	color: #ffffff;

}


QLabel::disabled
{
	background-color: transparent;
	color: #656565;

}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: transparent;
	color: #ffffff;

}


QLabel::disabled
{
	background-color: transparent;
	color: #656565;

}</string>
    </property>
    <property name="text">
     <string>Games Drawn</string>
    </property>
   </widget>
   <widget class="QLabel" name="drawsLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>110</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun</family>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel
{
	background-color: transparent;
	color: #ffffff;

}


QLabel::disabled
{
	background-color: transparent;
	color: #656565;

}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
